packages/dashboard/src/pages/CreateMaskWallet/CreateMnemonic/ComponentToPrint.tsx

```diff
@@ -0,0 +1,111 @@
+import { Box, Typography } from '@mui/material'
+import { makeStyles } from '@masknet/theme'
+import { QRCode } from 'react-qrcode-logo'
+import { PrintBackground } from '../../../assets/index.js'
+import { MnemonicReveal } from '../../../components/Mnemonic/index.js'
+import { Icons } from '@masknet/icons'
+import { NetworkType, networkResolver } from '@masknet/web3-shared-evm'
+
+interface ComponentToPrintProps {
+    words: string[]
+    address: string
+}
+
+const useStyles = makeStyles()((theme) => ({
+    container: {
+        display: 'flex',
+        justifyContent: 'center',
+        flexDirection: 'column',
+        padding: theme.spacing(6),
+        backgroundColor: theme.palette.maskColor.white,
+        width: 630,
+    },
+    card: {
+        width: '100%',
+        background: `url(${PrintBackground}) no-repeat`,
+        borderRadius: theme.spacing(1),
+        padding: theme.spacing(2),
+        color: theme.palette.maskColor.white,
+        display: 'flex',
+        alignItems: 'center',
+        backgroundSize: 'cover',
+    },
+    publicKeyTitle: {
+        fontSize: 14,
+        color: theme.palette.maskColor.white,
+        lineHeight: '18px',
+        fontWeight: 700,
+    },
+    publicKey: {
+        fontSize: 10,
+        color: theme.palette.maskColor.white,
+        lineHeight: '10px',
+    },
+    title: {
+        fontSize: 16,
+        lineHeight: '20px',
+        margin: theme.spacing(2.5, 0),
+        fontWeight: 700,
+    },
+    tips: {
+        marginTop: theme.spacing(4.5),
+        fontSize: 14,
+        lineHeight: '18px',
+        fontWeight: 700,
+        color: theme.palette.maskColor.publicMain,
+        display: 'flex',
+        alignItems: 'center',
+        columnGap: 12,
+    },
+    wordCard: {
+        backgroundColor: theme.palette.maskColor.publicBg,
+        color: theme.palette.maskColor.publicThird,
+        '&::marker': {
+            backgroundColor: theme.palette.maskColor.publicBg,
+            color: theme.palette.maskColor.publicThird,
+        },
+    },
+    text: {
+        color: theme.palette.maskColor.publicMain,
+    },
+}))
+
+export const ComponentToPrint = forwardRef((props: ComponentToPrintProps, ref: ForwardedRef<any>) => {
+    const { words, address } = props
+    const t = useDashboardI18N()
+    const { classes } = useStyles()
+
+    const qrValue = useMemo(() => {
+        return `mask://wallet/mnemonic/${btoa(words.join(' '))}`
+    }, [words?.join(',')])
+
+    return (
+        <Box className={classes.container} ref={ref}>
+            <Box className={classes.card}>
+                <Box flex={1}>
+                    <Typography className={classes.publicKeyTitle}>
+                        {t.address()}:{' '}
+                        <Typography component="span" className={classes.publicKey}>
+                            {address}
+                        </Typography>
+                    </Typography>
+                </Box>
+                <QRCode
```

Comment:
If no `words`, we should not present `QRCode`.

Reply:
`words: string[]` 
`words?` => `words`